tests/langchain/test_langchain_model_export.py

```diff
@@ -1142,6 +1142,53 @@ def fake_llm(prompt: str) -> str:
     }
 
 
+@pytest.mark.skipif(
+    Version(langchain.__version__) < Version("0.0.311"), reason="feature not existing"
+)
+def test_simple_chat_model_inference():
+    from langchain.chat_models.base import SimpleChatModel
+
+    class ChatModel(SimpleChatModel):
+        def _call(self, messages, stop, run_manager, **kwargs):
+            return "\n".join([f"{message.type}: {message.content}" for message in messages])
+
+        @property
+        def _llm_type(self) -> str:
+            return "chat model"
+
+    model = ChatModel()
+
+    with mlflow.start_run():
+        model_info = mlflow.langchain.log_model(model, "model")
+
+    loaded_model = mlflow.pyfunc.load_model(model_info.model_uri)
+
+    input_example = {
+        "messages": [
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "assistant", "content": "What would you like to ask?"},
+            {"role": "user", "content": "Who owns MLflow?"},
+        ]
+    }
+    expected_resp_content = {
+        "role": "assistant",
+        "content": (
+            "system: You are a helpful assistant.\n"
+            "ai: What would you like to ask?\n"
+            "human: Who owns MLflow?"
+        ),
+    }
+    response1 = loaded_model.predict([input_example])
+    assert len(response1) == 1
+    assert response1[0]["choices"][0]["message"] == expected_resp_content
+    response2 = loaded_model.predict(input_example)
+    assert response2["choices"][0]["message"] == expected_resp_content
+    response3 = loaded_model.predict([input_example, input_example])
+    assert len(response3) == 2
+    for i in range(2):
+        assert response3[i]["choices"][0]["message"] == expected_resp_content
+
```

Comment:
What's the stacktrace for this on `master` - i.e. how does it fail?

Reply:
On `master`, `SimpleChatModel` is `lc_runnables_types` , then the input list (e.g. [input1, input2]) is converted to `[[input1, input2]]` before passing to `process_api_requests`, so then in `APIRequest.call_api`, it is invoked with `[input1, input2]`, but the model is not `runnables_supports_batch_types`, so in `APIRequest.call_api`, it triggers `model.invoke([input1, input2])`, but `model.invoke` only supports single input, so error occurs.

So this is the `Fix 1` I described in PR description.